src/main/resources/db/migration/V1__init_database.sql

```diff
@@ -39,4 +40,11 @@ CREATE TABLE user_organisation (
     PRIMARY KEY (user_id, organisation_id),
     FOREIGN KEY (user_id) REFERENCES users(id),
     FOREIGN KEY (organisation_id) REFERENCES organisations(id)
+);
+
+-- V1__Create_waitlist_table.sql
```

Comment:
This should be in a new Flyway migration script with a unique version number.

Reply:
That is not my code oh, boss